app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,11 @@
 
   <body>
     <%= render "layouts/navbar" %>
+
+    <% flash.each do |type, message| %>
+      <% message = [message] if message.class == String %>
+        <%= content_tag :div, message.join('<br/>').html_safe, class: "alert alert-#{type}"%>
+    <%end%>
```

Comment:
Jmejia - This is to allow for custom flash messages sent in as strings, and also the logging of the errors object on a failed login which is returned as an array. Since we are calling message.join('<br/>') it will fail with a no method error if it comes in as a string, so I just convert the message to an array if this is the case.